tests/e2e/training-page-test.js

```diff
@@ -1,20 +1,21 @@
+import createServer from '../createServer'
 import { Selector, ClientFunction } from 'testcafe'
 require('dotenv').config()
 
-const { GATSBY_ENVIRONMENT } = process.env
-
-const baseUrl =
-  GATSBY_ENVIRONMENT === 'preview'
-    ? process.env.DEPLOY_PRIME_URL
-    : process.env.BASE_URL
-
+const baseUrl = `localhost:3001`
```

Comment:
could you break this like
```js
const hostname = 'localhost'
const port = 3001
const address = `${hostname}:${port}`

```

so that you can pass the port to `createServer`?